FF1Blazorizer/Tabs/TreasuresTab.razor

```diff
@@ -1,55 +1,58 @@
+	<div class="checkbox-cell"></div>
+	<h4>Trapped Chests</h4>
+	<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="trappedChestsCheckBox" @bind-Value="Flags.TrappedChests">Random Chests</TriStateCheckBox>
+	<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="masaGuardianCheckBox" @bind-Value="Flags.TCMasaGuardian">WarMech, Guardian of Masamune</TriStateCheckBox>
+	<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="trappedShardsCheckBox" IsEnabled="@Flags.ShardHunt" @bind-Value="Flags.TrappedShards">Trapped Shards</TriStateCheckBox>
+	<div class="checkbox-cell"></div>
+	<h4>Masamune</h4>
+	<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="GuaranteedMasamuneCheckBox" IsEnabled="@(Flags.NoMasamune != true)" @bind-Value="Flags.GuaranteedMasamune">Guaranteed Endgame Masamune</TriStateCheckBox>
+	<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Indent Id="SendMasamuneHomeCheckBox" IsEnabled="@((Flags.NoMasamune != true) && Flags.GuaranteedMasamune != false && Flags.IncentivizeMasamune == false)" @bind-Value="Flags.SendMasamuneHome">Send Masamune Home</TriStateCheckBox>
+	<div class="checkbox-cell"></div>
+</div>
+	<div class="col2">
+		<h4>Freebies</h4>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="freeShipCheckBox" @bind-Value="Flags.FreeShip">Free Ship</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="freeAirshipCheckBox" @bind-Value="Flags.FreeAirship">Free Airship</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="freeBridgeCheckBox" @bind-Value="Flags.FreeBridge">Free Bridge</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="freeCanalCheckBox" @bind-Value="Flags.FreeCanal">Free Canal</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="freeCanoeCheckBox" @bind-Value="Flags.FreeCanoe">Free Canoe</TriStateCheckBox>
+		<div class="checkbox-cell"></div>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="FreeLute" IsEnabled="@(!Flags.ShortToFR)" @bind-Value="Flags.FreeLuteFlag">Free Lute</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="FreeTail" IsEnabled="@(Flags.NoTail != true)" @bind-Value="Flags.FreeTail">Free Tail</TriStateCheckBox>
+		<CheckBox UpdateToolTip="@UpdateToolTipID" Id="freeOrbsCheckBox" IsEnabled="@Flags.FreeOrbsEnabled" @bind-Value="Flags.FreeOrbs">Free Orbs</CheckBox>
+		<div class="checkbox-cell"></div>
+		<h4>Bans</h4>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" IsEnabled="@(!Flags.FreeTail)" Id="NoTailCheckBox" @bind-Value="Flags.NoTail">Remove Tail</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="NoMasamuneCheckBox" @bind-Value="Flags.NoMasamune">Remove Masamune</TriStateCheckBox>
+		<div class="checkbox-cell"></div>
+		<h4>Consumables</h4>

```

Comment:
Should probably be put with Starting Inventory

Reply:
ok